spec/features/edit_beer_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+feature 'edit beer information', %Q{
+  As the original beer poster,
+  I want to be able to edit the information for the beer that I’ve added
+  so that people think I’m competent
+} do
+  # Acceptance Criteria
+  # * I must be signed in
+  # * I must be able to get here through the details page for a specific beer
+  # * I must fill in the required fields
+  # * I must be presented with errors if I fill out the form incorrectly
+
+  scenario 'sucessfully edited' do
+    user = FactoryGirl.create(:user)
+    beer = FactoryGirl.create(:beer, user: user)
+
+    visit root_path
+    click_link 'Sign In'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.